src/routes/trip.route.js

```diff
@@ -4,11 +4,13 @@ import oneWayTripValidation from '../validations/one-way-trip.validation';
 import authMiddleware from '../middlewares/auth.middleware';
 import TripValidation from '../validations/trip.validation';
 import requestValidation from '../validations/request.validation';
+import UserValidation from '../validations/user.validation';
 
 const router = express.Router();
 
 router.post('/one-way-trips', authMiddleware.checkUserLoggedIn, oneWayTripValidation, TripController.requestOneWayTrip); // One way trip route
 router.post('/return-trip', authMiddleware.checkUserLoggedIn, TripValidation, TripController.requestReturnTrip); // Return trip route
 router.get('/requests/:userId', authMiddleware.checkUserLoggedIn, requestValidation, TripController.userTripRequestList); // user request list route
+router.post('/comments', authMiddleware.checkUserLoggedIn, UserValidation.validateUserComment, TripController.userCommentOnRequestsTrip);
```

Comment:
Can you Please use `/comment/trip:tripId`. You should use the `tripId` in the `req.params` not `req.body`